### PR TITLE
Update Dockerfile and doc

### PR DESCRIPTION
**Pull request by xutaima**
Pass CLIENT_COMMAD to dockerfile instead of many env vars

### CODE REVIEW

File: examples/simultaneous_translation/docs/evaluation.md [hunk 1]
```diff
@@ -164,3 +164,20 @@ The latency metrics are
...
+```
+When submitting your final models, define a client command that would run inside the docker in a Dockerfile. For example, to evaluate the text translation in [baseline](baseline.md) experiment, the model can be evaluated as follow.
+
+
+```bash
+CLIENT_COMMAND="./examples/simultaneous_translation/scripts/start-multi-client.sh ./examples/simultaneous_translation/scripts/configs/must-c-en_de-text-dev.sh experiments/checkpoints/checkpoint_text_waitk3.pt"
+
+docker run --env CLIENT_COMMAND=$CLIENT_COMMAND -v "$(pwd)"/experiments:/fairseq/experiments -it iwslt2020_simulast
+```
+`CLIENT_COMMAND` can be the client command for customized client.
```
Comment (jmp84) [inline examples/simultaneous_translation/docs/evaluation.md:181]:
```suggestion
`CLIENT_COMMAND` can be the client command for a customized client.
```

File: examples/simultaneous_translation/docs/evaluation.md [hunk 2]
```diff
@@ -164,3 +164,20 @@ The latency metrics are
...
+
+When submitting you docker file, please keep the server settings in [example](../Dockerfile) and make sure it works for dev and open test set. We will modify the docker file for the blind test set.
```
Comment (jmp84) [inline examples/simultaneous_translation/docs/evaluation.md:183]:
```suggestion
When submitting you Docker file, please keep the server settings in [example](../Dockerfile) and make sure it works for dev and open test set. During the official eveluation, we will run the Docker file with different environment variables corresponding to the blind test sets.
```

**Comment by xutaima** [comment]
This pull request needs to be tested

File: examples/simultaneous_translation/docs/evaluation.md [hunk 3]
```diff
@@ -213,3 +213,20 @@ The latency metrics are
...
+## Final Evaluation with Docker
+Our final evaluation will be run inside Docker. To run an evaluation with Docker, first build a Docker image from the Dockerfile. Here is an [example](../Dockerfile) 
+```bash
+docker build -t iwslt2020_simulast:latest .
+```
+When submitting your final models, define a client command that would run inside the docker in a Dockerfile. For example, to evaluate the text translation in [baseline](baseline.md) experiment, the model can be evaluated as follow.
+
+
+```bash
+CLIENT_COMMAND="./examples/simultaneous_translation/scripts/start-multi-client.sh ./examples/simultaneous_translation/scripts/configs/must-c-en_de-text-dev.sh experiments/checkpoints/checkpoint_text_waitk3.pt"
```
Comment (jmp84) [inline examples/simultaneous_translation/docs/evaluation.md:226]:
```suggestion
CLIENT_COMMAND="./examples/simultaneous_translation/scripts/start-multi-client.sh ./examples/simultaneous_translation/scripts/configs/must-c-en_de-text-dev.sh experiments/checkpoints/checkpoint_text_waitk3.pt"
SRC_FILE=experiments/data/must_c_1_0/en-de/bi-text/tst-COMMON.en
TGT_FILE=experiments/data/must_c_1_0/en-de/bi-text/tst-COMMON.de
PORT=12321
```

File: examples/simultaneous_translation/docs/evaluation.md [hunk 4]
```diff
@@ -213,3 +213,20 @@ The latency metrics are
...
+
+docker run --env CLIENT_COMMAND=$CLIENT_COMMAND -v "$(pwd)"/experiments:/fairseq/experiments -it iwslt2020_simulast
```
Comment (jmp84) [inline examples/simultaneous_translation/docs/evaluation.md:228]:
```suggestion
docker run --env CLIENT_COMMAND="$CLIENT_COMMAND" -e SRC_FILE="$SRC_FILE" -e TGT_FILE="$TGT_FILE" -e PORT="$PORT" -v "$(pwd)"/experiments:/fairseq/experiments -it iwslt2020_simulast
```

**Pull request merged**